### PR TITLE
oem: avoid deprecated field when enabling systemd units

### PR DESCRIPTION
**Pull request by bgilbert**
Fixes warning:

    files: the enable field has been deprecated in favor of enabled

### CODE REVIEW

File: internal/oem/oem.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = &[]bool{true}[0]
```
Comment (dgonyeo) [inline internal/oem/oem.go]: Could you use `boolToPtr` in the util package?
https://github.com/coreos/ignition/blob/fe09ef5e14205d022765ebda59390bbee87fd36f/internal/util/util.go#L21

Comment (bgilbert) [inline internal/oem/oem.go]: Sure, if it's worth the additional import.  I checked and that function has zero callers.

**Comment by bgilbert** [comment]
Updated.

File: internal/oem/oem.go [hunk 2]
```diff
@@ -76,6 +77,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = util.BoolToPtr(true)
```
Comment (dgonyeo) [inline internal/oem/oem.go]: Maybe drop this line entirely and do `util.BoolToPtr(true)` everywhere below, but I'm rather indifferent about it.

LGTM.

Comment (bgilbert) [inline internal/oem/oem.go]: Yeah, it seemed cleaner to not have it open-coded all over the place.  Ready to merge from my end.

**Comment by bgilbert** [comment]
Rebased on latest.  Please merge.

**Pull request merged**